### PR TITLE
Object Cache Guide

### PR DESCRIPTION
**Pull request by whitneymeredith**
Closes # [DOC-686](https://getpantheon.atlassian.net/jira/software/projects/DOC/boards/186?selectedIssue=DOC-686)

## Summary

**[Object Cache Guide](https://pantheon.io/docs/doc-title)** - Break out the Object Cache doc into an easy-to-scan guide.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7689](https://pr-7689--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `46` | `49` | `3` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7689` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/41680/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/41680/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7689--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by whitneymeredith** [comment]
[Preview](https://pr-7689--pantheon-docs.my.pantheonfrontend.website/docs/guides/object-cache)

**Comment by whitneymeredith** [comment]
Redirect needed:
/docs/object-cache — > /guides/object-cache

File: source/content/guides/object-cache/03-redis-command-line.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+504 strings with 1403 bytes (99.60% of keys, avg size 2.78)
+1 lists with 100004 items (00.20% of keys, avg size 100004.00)
+0 sets with 0 members (00.00% of keys, avg size 0.00)
+1 hashs with 3 fields (00.20% of keys, avg size 3.00)
+0 zsets with 0 members (00.00% of keys, avg size 0.00)
+```
+
+
+## More Resources
```
Comment (jms-pantheon) [inline source/content/guides/object-cache/03-redis-command-line.md:186]: These resources aren't relevant to the page, not that we're using a local "terminal" here but not the Pantheon "Terminus" tool. The other, this is the Redis command line interface (CLI) but does not relate to the WordPress CLI

File: source/content/guides/object-cache/04-remove-object-cache.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: Object Cache (formerly Redis)
+subtitle: Remove Object Cache
+description: Learn how to safely remove Object Cache.
+categories: [performance]
+tags: [cache, plugins, modules, database]
+contributors: [cityofoaksdesign, carolynshannon, whitneymeredith]
+layout: guide
+showtoc: true
```
Comment (jms-pantheon) [inline source/content/guides/object-cache/04-remove-object-cache.md:9]: Could drop TOC here too, just the one item

File: source/content/guides/object-cache/05-troubleshoot-object-cache.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    used_memory_rss_human:1.91M
+    used_memory_peak:1008128
+    used_memory_peak_human:984.50K
+  ```
+
+## More Resources
+
+- [Basic Troubleshooting](/basic-troubleshooting)
+- [PHP Errors and Exceptions](/guides/php/php-errors)
+- [site.env](/terminus/examples#siteenv)
```
Comment (jms-pantheon) [inline source/content/guides/object-cache/05-troubleshoot-object-cache.md:137]: This Terminus command isn't used in this guide, I suggest removing it

**Review by jms-pantheon — CHANGES_REQUESTED** [review]
Just a few changes to layout and More Resources, thanks!

**Comment by wordsmither** [comment]
lgtm

**Comment by jms-pantheon** [comment]
@whitneymeredith should the old long doc be removed and redirected as part of this?

**Pull request merged**